components/DatasetDetails/DatasetAboutInfo.vue

```diff
@@ -4,7 +4,9 @@
       <h3>Last Updated</h3>
       <p>{{ updatedDate }}</p>
       <h3>Dataset DOI</h3>
-      <p>{{ doi }}</p>
+      <div class="dataset-about-info__container--doi-link mb-16">
```

Comment:
You're extending `p` in Sass, but why not keep it a `p`? Having it stay a `p` would keep it somewhat semantic markup.

Reply:
Ah, right. Updated per your suggestions.